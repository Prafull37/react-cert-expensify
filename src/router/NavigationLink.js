import React from "react";
import {NavLink} from 'react-router-dom'

const NavigationLink=(props)=>(
    <div>
    <NavLink to="/" exact activeClassName="is-active">Dashborad</NavLink>
    <NavLink to="/create" activeClassName="is-active">Add Expense</NavLink>
</div>
)

export default NavigationLink;