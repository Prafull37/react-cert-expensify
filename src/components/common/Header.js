import React from 'react';
import NavigationLink from "../../router/NavigationLink";

const Header=(props)=>(
    <header>
        <h2>Expensify</h2>
        <NavigationLink/>
    </header>
)

export default Header;