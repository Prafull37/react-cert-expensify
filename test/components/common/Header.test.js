import React from "react";
import {shallow} from "enzyme";
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from "../../../src/components/common/Header";
import NavigationLink from "../../../src/router/NavigationLink"

test("test NavigationLink by snapshot",()=>{
   const wrapper=shallow(<Header/>)
   expect(wrapper).toMatchSnapshot();
})