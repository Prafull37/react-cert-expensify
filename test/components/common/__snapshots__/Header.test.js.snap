// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test NavigationLink by snapshot 1`] = `
<header>
  <h2>
    Expensify
  </h2>
  <NavigationLink />
</header>
`;
